src/platforms/php/common/performance.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Performance
+description: "Learn more about how to configure our Performance integrations to get the best experience out of it."
+---
+
+: `traces_sampler`
+
+If you want to sample specific Transaction, use the `traces_sampler` option:
+
+```php
+Sentry\init([
+    'dsn' => '___PUBLIC_DSN___',
+    'traces_sampler' => static function (Sentry\Tracing\SamplingContext $context): float {
+        // This example will discard all Transaction that have health in their name
+        if (false === strpos($context->getTransactionContext()->getName(), 'health')) {
+            return 1.0;
+        }
+        return 0;
+    },
+]);
+```
+
+### Manual Instrumentation
```

Comment:
Is there no automatic instrumentation for Laravel?

Reply:
There is; Good catch I will document what we automatically track.